legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/mapping/MappingLSPGrammarExtension.java

```diff
@@ -264,6 +250,74 @@ private List<? extends LegendExecutionResult> runLegacyMappingTests(SectionState
         return results;
     }
 
+    private List<? extends LegendExecutionResult> analyzeMappingModelCoverage(SectionState section, String entityPath)
+    {
+        PackageableElement element = getParseResult(section).getElement(entityPath);
+        CompileResult compileResult = this.getCompileResult(section);
+
+        if (compileResult.hasException())
+        {
+            return Collections.singletonList(errorResult(compileResult.getCompileErrorResult(), entityPath));
+        }
+
+        try
+        {
+            if (this.isEngineServerConfigured()
```

Comment:
I will avoid this if, as this is mostly static code, and does not required authentication to run, hence running always locally would make sense.

Function execution is one of the few things we dispatch to the server as connecting to some Databases is not possible yet in LSP.

Reply:
Makes sense. I wasn't sure if we wanted all functionality to be able to be done remotely, so I added this. But if we don't need to support everything remotely, then we don't need this, so I removed it.